packages/insomnia-components/components/list-group/unit-test-result-badge.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+
+type Props = {|
+  result?: String,
```

Comment:
I suggest changing this prop to be more explicit

```suggestion
  failed?: boolean,
```